atest/regexp_prompt.robot

```diff
@@ -10,15 +10,10 @@ Read Until Prompt With Regexp Prompt
     Write  pwd
     ${output}=	Read Until Prompt
     Should Contain	${output}  $
-    Write  su test_prompt
-    ${password}=	Read	delay=0.5s
-    Should Contain	${password}	Password:
-    Write  test
+    Write  export PS1\='\\u\@\\h \\W # '
     ${output1}=	Read Until Prompt
```

Comment:
Good, but there is a catch. I think Read until prompt will match the # in `export PS1` command, instead of the prompt. So output will be:
```
export PS1='\u@\h \W # '
test@host ~ # 
```
but read until prompt will read only read `export PS1='\u@\h \W #`. After export command need to consume output with `Read` to not match the command itself.